bootstrap.sh

```diff
@@ -5,18 +5,40 @@ cd $(dirname "$BASH_SOURCE")
 set -ev
 
 # Update puppet to version >= 3.2.2 before using puppet provisioning.
-wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-dpkg -i puppetlabs-release-pc1-trusty.deb
+if [ ! -e puppetlabs-release-pc1-trusty.deb ]
+then
+    wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
+fi
+dpkg --install puppetlabs-release-pc1-trusty.deb
 apt-get update
-apt-get -y install puppet-agent
+apt-get --assume-yes install puppet-agent
 export PATH=/opt/puppetlabs/bin:$PATH
 
-cp -f Puppetfile /opt/puppetlabs/puppet/Puppetfile
+DIRECTORY=/opt/mes-aides-bootstrap
+mkdir --parents $DIRECTORY/manifests
+for element in bootstrap ops
+do
+    REPO_PATH=manifests/$element.pp
+    if [ -e $REPO_PATH ]
+    then
+        cp $REPO_PATH $DIRECTORY/$REPO_PATH
+    else
+        wget --output-document=$DIRECTORY/$REPO_PATH https://raw.githubusercontent.com/sgmap/mes-aides-ops/master/$REPO_PATH
+    fi
+done
 
-cd /opt/puppetlabs/puppet/
-gem install librarian-puppet
-librarian-puppet install
+# One off script that
+# * install librarian-puppet in Puppet ruby to download Puppet modules
+# * download a bootstrap Puppetfile
+# * download specified modules
+puppet apply /opt/mes-aides-bootstrap/manifests/bootstrap.pp --verbose --modulepath=modules
 
-cd -
-cp -r modules/mesaides /opt/puppetlabs/puppet/modules/
-/opt/puppetlabs/bin/puppet apply manifests/default.pp --verbose --modulepath=/opt/puppetlabs/puppet/modules
+# Script to run on mes-aides-ops update
+# * update local mes-aides-ops repository
+# * download modules
+puppet apply /opt/mes-aides-bootstrap/manifests/ops.pp --verbose --modulepath=/opt/mes-aides-bootstrap/modules
```

Comment:
Do you really need to run those manually here? Shouldn't it be bootstrap's responsibility to start ops? And ops' responsibility to start applicative install?